tools/emacs/soutei-mode.el

```diff
@@ -0,0 +1,49 @@
+(defvar soutei-mode-hook nil)
+
+(defface soutei-keyword-face
+  '((t (:inherit font-lock-keyword-face)))
+  "Soutei keywords")
+
+(defface soutei-operator-face
+  '((t (:inherit font-lock-builtin-face)))
+  "Soutei builtin operators")
+
+(defvar soutei-indent-width prolog-indent-width
```

Comment:
I think you might need to `(require 'prolog-mode)` before `prolog-indent-width` becomes available?

Reply:
Also, I've been using 2 across the board---I'm torn between not wanting to be overly prescriptive and possibly establishing a slightly stronger style convention from the start.  Thoughts on not letting `prolog-mode` dictate things here?

Reply:
`prolog-mode` is apparently built in to emacs, and I just tested with `emacs -q` that it's available without `(require 'prolog)`. I'm fine imposing *your* style preference, just didn't want to impose mine. :)

Reply:
I got an unbound-variable error on `prolog-indent-width` without the `require`.

Reply:
The plot thickens.  Is it still an issue now that we're deriving from `prolog-mode` but not using that variable?

Reply:
Seems to work for me now.  For a good time, try `(setq prolog-electric-dot-flag t prolog-electric-colon-flag t)` :)